graylog2-web-interface/src/components/common/Select.jsx

```diff
@@ -4,16 +4,26 @@ import lodash from 'lodash';
 import PropTypes from 'prop-types';
 
 import ReactSelect, { components as Components, Creatable } from 'react-select';
-// eslint-disable-next-line import/no-webpack-loader-syntax
-import reactSelectSmStyles from '!style/useable!css!./Select.css';
-
-const MultiValueRemove = (props) => {
-  return (
-    <Components.MultiValueRemove {...props}>
-      ×
-    </Components.MultiValueRemove>
-  );
-};
+
+const MultiValueRemove = props => (
+  <Components.MultiValueRemove {...props}>
+    ×
```

Comment:
Should we use the HTML encoded `&times;`?

Reply:
✅